{{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/browser/configure.zcml

```diff
@@ -13,7 +13,7 @@
 
   <!-- Publish static files -->
   <browser:resourceDirectory
-      name="{{cookiecutter.project_slug}}"
+      name="{{cookiecutter.package_name}}"
```

Comment:
Not a fan of having dots in the resourcedirectories names. I found several issues when using them.